docs/tutorials/Astro/deploying-hybrid-projects.md

```diff
@@ -0,0 +1,286 @@
+import apostrophe from '@apostrophecms/apostrophe-astro';
+
+export default defineConfig({
+  output: "server",  // Required for SSR
+  server: {
+    port: process.env.PORT ? parseInt(process.env.PORT) : 4321,
+    // Uncomment for hosting platforms like Heroku that need this
+    // host: true
+  },
+  adapter: node({
+    mode: 'standalone'  // For most deployment scenarios
+  }),
+  integrations: [apostrophe({
+    // In production, this should be set via the APOS_HOST env variable
+    aposHost: process.env.APOS_HOST || 'http://localhost:3000',
+    widgetsMapping: './src/widgets',
+    templatesMapping: './src/templates',
+    
+    // Security headers to pass from backend to frontend
+    includeResponseHeaders: [
+      'content-security-policy',
+      'strict-transport-security',
+      'x-frame-options',
+      'referrer-policy',
+      'cache-control'
+    ],
+    
+    // For split deployment (separate servers), you may need to uncomment:
+    // excludeRequestHeaders: ['host']
+  })],
+  
+  // Required to handle virtual URLs in the integration
+  vite: {
+    css: {
+      preprocessorOptions: {
+        scss: {
+          quietDeps: true
+        }
+      }
+    },
+    ssr: {
+      noExternal: ['@apostrophecms/apostrophe-astro']
+    }
+  },
+  
+  // SCSS configuration if using Sass
+  css: {
+    preprocessorOptions: {
+      scss: {
+        api: 'modern-compiler',
+      }
+    }
+  }
+});
+```
+
+### Key Configuration Areas
+
+1. **Port Configuration**
+   - The `server.port` setting defaults to 4321 but reads from the `PORT` environment variable if set
+   - Some platforms (Heroku, Railway) require `host: true` to listen on all interfaces
+
+2. **Integration Settings**
+   - `aposHost` must point to your production backend URL in production
+   - Set via the `APOS_HOST` environment variable rather than hardcoding
+
+3. **Header Configuration**
+   - `includeResponseHeaders` determines which response headers from ApostropheCMS are preserved
+   - Essential for maintaining security settings between backend and frontend
+
+4. **Split Deployment Settings**
+   - When deploying to separate servers, you may need to exclude the `host` header
+   - Uncomment `excludeRequestHeaders: ['host']` to prevent hostname conflicts
+
+
+## Deploying with Apostrophe Hosting
+
+Apostrophe offers a straightforward hosting solution specifically designed for ApostropheCMS projects, including those with Astro frontends.
+
+### Benefits
+
+- Zero configuration for ApostropheCMS + Astro integration
+- Database provisioning and management handled automatically
+- Built-in asset storage and delivery
+- SSL certificate management
+- Automatic backups
+- Security updates
+
+### Getting Started with Apostrophe Hosting
+
+Contact the Apostrophe team through their [website](https://apostrophecms.com/contact-us) to set up your hosting. The team will provide detailed instructions for connecting your repository for automatic deployment.
```

Comment:
Please link to our hosting page.

Reply:
( Not customer portal, we have a hosting landing page: https://apostrophecms.com/hosting )

Reply:
Fixed.